lib/redlock/client.rb

```diff
@@ -233,14 +242,22 @@ def try_lock_instances(resource, ttl, options)
       retry_count = options[:retry_count] || @retry_count
       tries = options[:extend] ? 1 : (retry_count + 1)
 
+      last_error = nil
+
```

Comment:
I like whitespaces for logical grouping, but in this case empty line before `last_error` declaration seems excessive